src/components/ArticlePage.jsx

```diff
@@ -1,27 +1,62 @@
 import React from 'react';
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { fetchSingleArticle } from '../ApiRequests';
+import { fetchSingleArticle,fetchArticleComments } from '../ApiRequests';
+import './ArticlePage.css';
 
+// complete is error state 
 
 function ArticlePage() {
   const { article_id } = useParams(); 
-  console.log(article_id,"aRTICLEPage")
-    const [article, setArticle] = useState({})
+ const [isCommentsVisible, setCommentsVisible] = useState(false)
+  const [article, setArticle] = useState({})
+  const [comments, setComments] = useState([])
+  const [isLoading, setIsLoading] = useState(true)
+ 
+  
     useEffect(() => {
-      fetchSingleArticle(article_id).then((articleFromApi) => {
-        console.log("article from api", articleFromApi);
-        setArticle(articleFromApi);
+      fetchSingleArticle(article_id)
+        .then((articleFromApi) => {
+        setArticle(articleFromApi)
+        setIsLoading(false)
       });
-    }, {});
+    }, []);
+  
+  useEffect(() => {
+    fetchArticleComments(article_id)
+      .then((commentsFromApi) => {
+     setComments(commentsFromApi);
+   
+   });
+  }, []);
+  
+  if (isLoading) {
+    return <div>Is Loading....</div>;
+  }
+  
   return (
     <div>
-          <p>{article.title}</p>
-          <p>{article.author}</p>
-          <p>{article.created_at}</p>
-          <p>{article.topic}</p>
-          <p>{article.comment_count}</p>
-          <p>{article.votes}</p>
+      <p>{article.title}</p>
+      <p>{article.author}</p>
+      <p>{article.created_at}</p>
+      <p>{article.topic}</p>
+      <p>{article.votes}</p>
+      <button onClick={() => setCommentsVisible(!isCommentsVisible)}>
+        <p>{article.comment_count} comments </p>
+      </button>
+      {isCommentsVisible ? (
+        <div>
+          All comments
+          {comments.map((comment, index) => (
+            <div>
+              <p>{comment.author}</p>
+              <p>{comment.created_at}</p>
+              <p>{comment.votes}</p>
+              <p>{comment.body}</p>
```

Comment:
This ArticlePage component is a little bloated as it contains the code not only for the article but also the comments, I think it would be a good idea to split the code for the comments out to it's own component that you can slot into the `ArticlePage`, this would also mean that you won't have two `useEffects` going on in the same component